Modules/LDAPObject.h

```diff
@@ -47,4 +47,13 @@ extern LDAPObject *newLDAPObject(LDAP *);
 	  PyEval_RestoreThread( _save );                                \
 	}
 
+#define BEGIN_ALLOW_THREADS                                         \
+    do {                                                            \
+        PyThreadState *_save;                                       \
+        _save = PyEval_SaveThread()                                 \
+
+#define END_ALLOW_THREADS                                           \
+        PyEval_RestoreThread(_save);                                \
+    } while(0)
+
```

Comment:
This isn't something I'd want to maintain in python-ldap.

I think you can follow Python's `Py_*_ALLOW_THREADS` macros with a semicolon. Or is there some reason you can't?
(If not, I'd rather propose that in Python, and possibly keep a backport here for current/past Python versions.)